.gitmodules

```diff
@@ -228,6 +228,15 @@
 [submodule "contrib/yaml-cpp"]
 	path = contrib/yaml-cpp
 	url = https://github.com/ClickHouse-Extras/yaml-cpp.git
+[submodule "contrib/libstemmer_c"]
```

Comment:
I think we should move these libs to ClickHouse-Extras, but please, can you add some instructions to your repos: Where did you get them?

Reply:
Updated READMEs.